utils/vlog/vlog_to_model.py

```diff
@@ -12,15 +12,18 @@
     ".subckt name" in the BLIF model. Mostly intended for use with w.py, when several
     different pb_types implement the same model.
 
+    - `(* CLASS="lut|routing|mux|flipflop|mem" *)` : specify the class of an given
+    instance. A model will not be generated for the `lut`, `routing` or `flipflop`
+    class.
```

Comment:
Are these attributes common between `vlog_to_model` and `vlog_to_pbtype`?

Reply:
Yes